core/tests/webdriverio_utils/action.js

```diff
@@ -20,7 +20,7 @@ var waitFor = require('./waitFor.js');
 
 // Waits for the invisibility of the autosave message.
 var waitForAutosave = async function() {
-  var autoSaveIndicatorElement = $('.e2e-test-autosave-indicator');
+  var autoSaveIndicatorElement = await $('.e2e-test-autosave-indicator');
```

Comment:
Will this not create that [read ECONNRESET](https://github.com/oppia/oppia/runs/7498708295?check_suite_focus=true#step:13:523) flake too?

Reply:
Removed.

Reply:
My comment would still remain the same. Because it follow's the same pattern as we have for `pageToLoadFully`.

Reply:
We actually uses the same pattern everywhere, pageToReload is only causing flake which I am debugging else everywhere we do not need await.